docs/tutorials/gluon/logistic_regression_explained.md

```diff
@@ -120,96 +108,115 @@ for e in range(epochs):
+    Epoch: 7, Training loss: 56.08, Validation loss: 10.59, Validation accuracy: 0.96 <!--notebook-skip-line-->
 
-## Tip 3: Use SigmoidBinaryCrossEntropyLoss instead of LogisticRegressionOutput
+    Epoch: 8, Training loss: 56.10, Validation loss: 6.74, Validation accuracy: 0.97 <!--notebook-skip-line-->
 
-NDArray API has two options to calculate logistic regression loss. One is `SigmoidBinaryCrossEntropyLoss`, which I used in the example. This class inherits from the `Loss` class and is intended to be used as a loss function for logistic regression. But there is also a function called `LogisticRegressionOutput`, which can be applied to any `NDArray`. Mathematically speaking, this function does the same thing as `SigmoidBinaryCrossEntropyLoss`. 
+    Epoch: 9, Training loss: 51.81, Validation loss: 7.48, Validation accuracy: 0.98 <!--notebook-skip-line-->
 
-My recommendation would be to use `SigmoidBinaryCrossEntropyLoss`, because this class properly inherits from `Loss` class, while `LogisticRegressionOutput` is just a regular function. `LogisticRegressionOutput` is a   function to go when implementing logistic regression using Symbol API, but in case of using Gluon API, there are no benefits using it. The only case when you may want to consider using `LogisticRegressionOutput` as your loss, is when you need to have a support for sparse matrices.
 
-## Tip 4: Convert probabilities to classes before calculating Accuracy
+In our case we easily hit the accuracy of 0.98.
 
-`Accuracy` metric requires 2 arguments: 1) a vector of ground-truth classes and 2) A tensor of predictions. When tensor of predictions is of the same shape as the vector of ground-truth classes, `Accuracy` class assumes that it should contain predicted classes. So, it converts the vector to `Int32` and compare each item of ground-truth classes to prediction vector. 
+## Tip 1: Use only one neuron in the output layer
 
-Because of the behaviour above, you will get an unexpected result if you just pass the output of `Sigmoid` function as is. `Sigmoid` function produces output in range [0; 1], and all numbers in that range are going to be casted to 0, even if it is as high as 0.99. To avoid this we write a custom bit of code, that:
+Despite that there are 2 classes, there should be only one output neuron, because `SigmoidBinaryCrossEntropyLoss` accepts only one feature as an input. 
 
-1. Subtracts a threshold from the original prediction. Usually, the threshold is equal to 0.5, but it can be higher, if you want to increase certainty of an item to belong to class 1.
+In case when there are 3 or more classes, one cannot use a single Logistic regression, but should do multiclass regression. The solution would be to increase the number of output neurons to the number of classes and use `SoftmaxCrossEntropyLoss`. 
```

Comment:
* "In case there are".
* Multi-class regression refers to the case when one item belongs to multiple classes. In that case you cannot use softmax, but rather multiple logistic regressions, one for each class. I suggest to simply remove this sentence to not create any confusion.

Reply:
Removed